source/api_cc/src/DeepPotJAX.cc

```
@@ -0,0 +1,779 @@
+  // (first three and last four)
+  std::array<std::uint8_t, 19> config = {
+      {0x10, static_cast<std::uint8_t>(num_intra_nthreads), 0x28,
+       static_cast<std::uint8_t>(num_inter_nthreads), 0x32, 0xb, 0x9, 0xFF,
+       0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0x20, 0x1, 0x38, 0x1}};
+
+  // Convert the desired percentage into a byte-array.
+  auto bytes = reinterpret_cast<std::uint8_t*>(0.9);
+
+  // Put it to the config byte-array, from 3 to 10:
+  for (std::size_t i = 0; i < sizeof(gpu_memory_fraction); ++i) {
+    config[i + 3] = bytes[i];
```

cppcheck

[error] Null pointer dereference

(nullPointer)